src/addonmanager.cpp

```diff
@@ -258,20 +257,23 @@ void AddonManager::unload(const QString& addonId) {
 
   Addon* addon = m_addons[addonId].m_addon;
 
-  if (addon) {
```

Comment:
All of these changes are not part of a separate PR: https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4071